src/Microsoft.AspNetCore.Blazor.Build/Core/RazorCompilation/RazorCompiler.cs

```diff
@@ -96,7 +96,7 @@ public class RazorCompiler
                 var engine = new BlazorRazorEngine();
                 var blazorTemplateEngine = new BlazorTemplateEngine(
                     engine.Engine,
-                    RazorProject.Create(inputRootPath));
+                    RazorProjectFileSystem.Create(inputRootPath));
```

Comment:
`RazorProject` has been obsoleted.

Deriving for `RazorTemplateEngine` has also been deprecated, and is in the process of being phased out from our repos. I have these code changes at home, I'll push them later as a follow up.